src/leaderboard.py

```diff
@@ -0,0 +1,106 @@
+# TODO: add docstrin here
+import time
+import logging
+import traceback
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+import praw
+
+import config
+import utils
+import formula
+import message
+from kill_handler import KillHandler
+from models import Investor, Firm
+from stopwatch import Stopwatch
+
+logging.basicConfig(level=logging.INFO)
+
+sidebar_text_org = """
+test
+123
+
+noerdy should put stuff here
+
+TODO
+
+===================
+(^ is that how you make horizontal rules?)
+
+Top Users:
+%TOP_USERS%
+
+Top Firms:
+%TOP_FIRMS%
+"""
+
+# TODO: add docstring
+def main():
+    logging.info("Starting leaderboard...")
+    logging.info("Sleeping for 8 seconds. Waiting for the database to turn on...")
+    time.sleep(8)
+
+    killhandler = KillHandler()
+
+    engine = create_engine(config.DB, pool_recycle=60, pool_pre_ping=True)
+    session_maker = sessionmaker(bind=engine)
+
+    reddit = praw.Reddit(client_id=config.CLIENT_ID,
+                         client_secret=config.CLIENT_SECRET,
+                         username=config.USERNAME,
+                         password=config.PASSWORD,
+                         user_agent=config.USER_AGENT)
+
+    # We will test our reddit connection here
+    if not utils.test_reddit_connection(reddit):
+        exit()
+
+    while not killhandler.killed:
+        sess = session_maker()
+
+        top_users = sess.query(Investor).\
+            order_by(Investor.balance.desc()).\
```

Comment:
We can sort by balance or net worth. Net worth is basically just balance + all active investments. So if I have 1mil mc, I invest all of them and my balance uis 0mc. If you want to, you can sort by using net worth, we have a piece of code that does that in the api module.